src/cats/CatSlideshowControls.js

```diff
@@ -0,0 +1,27 @@
+import styles from "./CatSlideshowControls.module.css";
+
+export default function CatSlideshowControls({
+  dispatch,
+  isDisabled,
+  canScrollLeft,
+  canScrollRight,
+}) {
+  return (
+    <div className={styles.catSlideshowControls}>
+      <button
+        className={styles.button}
+        disabled={isDisabled || !canScrollLeft}
+        onClick={() => dispatch({ type: "decrement-index" })}
+      >
+        &lt;
+      </button>
+      <button
+        className={styles.button}
+        disabled={isDisabled || !canScrollRight}
+        onClick={() => dispatch({ type: "increment-index" })}
```

Comment:
I ended up replacing this code so that a function wrapped is useCallback comes from CatSlideshow and is passed unchanged to the button elements. I think that accomplishes the same thing.

Reply:
Worth noting there is no benefit to memoizing callbacks passed to raw elements -- React does the same amount of work either way